kolibri/plugins/learn/assets/src/vue/toolbar/index.vue

```diff
@@ -1,22 +1,14 @@
 <template>
 
-  <div v-bind:class="['toolbar-show', displayToolBar ? 'toolbar-hide' : '' ]" v-show='!searchOpen' >
+  <div v-bind:class="['toolbar', displayToolbar ? 'toolbar-hide' : '']" v-show='!searchOpen'>
     <breadcrumbs class="breadcrumbs"></breadcrumbs>
-    <div :class="{ 'toggle-menu-on' : more }">
-      <label for="chan-select" :class="[ more ? 'label-on' : 'visuallyhidden' ]" >{{ $tr('switchChannels') }}</label>
```

Comment:
I must have been doing a banged up job so far, if you had to google it... :worried: 

It had the global `visuallyhidden` class so far, so you can apply it again!